ruby_programming/testing_with_rspec/introduction_to_rspec.md

```diff
@@ -0,0 +1,236 @@
+# Introduction to RSpec
+
+## Introduction
+In the previous lesson, we established the utility of [test-driven development](https://en.wikipedia.org/wiki/Test-driven_development) (TDD) in maintaining your code and sanity. In this lesson, we'll introduce you to your new best friend, the [RSpec](http://rspec.info/) testing framework. RSpec is one of the most popular testing frameworks, having been downloaded over [100 million times](https://rubygems.org/gems/rspec), as of this writing, and having been ported for use in [Rails testing](https://rubygems.org/gems/rspec-rails).
+
+## Learning Outcomes
```

Comment:
Can we change all the h2 headings denoted with `##` to h3's please?  The styling on the main site expects h3's for any top level headings in lesson content.